packages/MainUI/components/Forms/GridLayout.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, Paper, Typography } from '@mui/material';
+import { useStyle } from './styles';
+import { GRID_CONSTANTS } from './constants';
+import EtendoImg from '../../../ComponentLibrary/src/assets/images/Etendo.svg?url';
+import { GridItemProps } from './types';
+import { useTranslation } from '../../hooks/useTranslation';
+
+const GridItem = ({ bgColor = 'transparent', children }: GridItemProps) => {
+  const { styles } = useStyle();
+
+  return (
+    <Paper
+      elevation={8}
+      sx={{
+        ...styles.gridItem,
+        backgroundColor: bgColor,
+        ...(bgColor !== 'transparent' && styles.gridItemWithBg),
+      }}>
+      <Box sx={styles.gridItemContent}>{children}</Box>
+    </Paper>
+  );
+};
+
+const GridLayout = () => {
+  const { styles } = useStyle();
+  const { ITEMS } = GRID_CONSTANTS;
```

Comment:
if grid constants is constant (what a discover!) then you may declare this outside of the component
```tsx

const { ITEMS } = GRID_CONSTANTS;

const GridLayout = () => {
.
.
.

```